### DISCUSSION TITLE
E1766 as a on/off switch

### DISCUSSION THREAD

**Discussion by ilbarone87** [Koenkk/zigbee2mqtt#18133] [Devices]
Is it possible to reconfigure the E1766 to be used as a light switch for on/off?

Thanks

**Comment by aatifk** [comment]
I'd be curious to know that as well. A rudimentary solution that I've found is to use [Awesome HA Blueprints](https://epmatt.github.io/awesome-ha-blueprints/docs/controllers-hooks-ecosystem) to set up automations, but they don't work as effectively as a native binding would.
